glue/src/it/scala/com/gu/tableversions/glue/GlueMetastoreSpec.scala

```diff
@@ -0,0 +1,115 @@
+package com.gu.tableversions.glue
+
+import java.net.URI
+
+import cats.effect.IO
+import cats.implicits._
+import com.amazonaws.auth.{
+  EnvironmentVariableCredentialsProvider,
+  InstanceProfileCredentialsProvider,
+  SystemPropertiesCredentialsProvider,
+  _
+}
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.services.glue.model._
+import com.amazonaws.services.glue.{AWSGlue, AWSGlueClient}
+import com.gu.tableversions.core.Partition.PartitionColumn
+import com.gu.tableversions.core._
+import com.gu.tableversions.metastore.MetastoreSpec
+import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
+
+import scala.util.{Properties, Random}
+
+class GlueMetastoreSpec extends FlatSpec with Matchers with BeforeAndAfterAll with MetastoreSpec {
+
+  def readMandatoryEnvVariable(varName: String) =
+    Properties.envOrNone(varName).toRight(s"$varName environment variable must be set")
+
+  val AWSProfileEnvVarName = "TABLE_VERSIONS_TEST_AWS_PROFILE"
+  val AWSRegionEnvVarName = "TABLE_VERSIONS_TEST_AWS_REGION"
+  val SchemaEnvVarName = "TABLE_VERSIONS_TEST_GLUE_DATABASE"
+
+  val maybeAwsProfile = Properties.envOrNone(AWSProfileEnvVarName)
+  val maybeSchema = Properties.envOrNone(SchemaEnvVarName)
+
+  val envVars = for {
+    schema <- readMandatoryEnvVariable(SchemaEnvVarName)
+    awsRegion <- readMandatoryEnvVariable(AWSRegionEnvVarName)
+    awsProfile = Properties.envOrNone(AWSProfileEnvVarName)
+  } yield (schema, awsRegion, awsProfile)
+
+  envVars match {
+    case Left(error)                                 => cancel(error)
+    case Right((schema, awsRegion, maybeAwsProfile)) => runWithVariables(schema, awsRegion, maybeAwsProfile)
```

Comment:
I would normally make the dev params the default values if the env variables are not found, but I got the sense we want to remove all guardian specific references from the code so the test is aborted if the variables are not there.